.github/workflows/workflow.yml

```diff
@@ -32,6 +32,28 @@ jobs:
     - name: Verify no unstaged changes
       if: runner.os != 'windows'
       run: __tests__/verify-no-unstaged-changes.sh
+  
+  test-setup-multiple-versions:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system: [ubuntu-latest, windows-latest, macOS-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Clear toolcache
+        shell: pwsh
+        run: __tests__/clear-toolcache.ps1 ${{ runner.os }}
+      - name: Setup dotnet 2.2.402 and 3.1.404
+        uses: ./
+        with:
+          dotnet-version: |
```

Comment:
Could you please add version with wildcard to this testcase? For example:
```
with:
   dotnet-version: |
     2.2.402
     3.1.404
     5.0.x

```

Reply:
Maybe it would be nice to add this to the README example as well

(and maybe show that it also works with `include-prerelease`?)

Reply:
Resolved